Integrations/integration-EWSv2.yml

```diff
@@ -585,6 +585,7 @@ script:
             return result
 
         def call(self, query, mailboxes):
+            query = query.replace(".", "\\.")
```

Comment:
how all this lines of code related to release? I thought we just remove the releaseNotes here